appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java

```diff
@@ -159,6 +166,36 @@ public VmApiProxyDelegate() {
           "complete. Service bridge status code: %s; response " +
           "content-length: %s. Took %s ms.", packageName, methodName, requestData.length, 200,
           responseData.length, (end - start)));
+      
+      // HACK TO FIX USER_SERVICE BUG
+      if ("user".equals(packageName)) {
+        String host = (String) environment.getAttributes().get("com.google.appengine.runtime.host");
+        String https = (String) environment.getAttributes().get("com.google.appengine.runtime.https");
+        if (host != null && host.length() > 0) {
```

Comment:
@ludoch I added those attributes to the AttributeMapping enum in VmApiProxyEnvironment